src/NHibernate/Driver/Sql2008ClientDriver.cs

```diff
@@ -2,19 +2,21 @@
 using System.Data;
 using System.Data.Common;
 using System.Data.SqlClient;
-using System.Linq;
 
 namespace NHibernate.Driver
 {
 	public class Sql2008ClientDriver : SqlClientDriver
 	{
+		public const byte MaxTime = 5;
```

Comment:
Why is this 5? It shall be either 7 or 8 (same as datetime2)

Reply:
Yes, the same for `datetime2` and `datetimeoffset`, where it wants to knows their size instead of their  scale.

Reply:
About those `MaxSizes`, I have put it public for mimicking what was done for others. But well, I do not think there is need for that. Maybe time to clean-up a bit the driver by switching back to protected those parameter sizing members.